x-pack/test/security_solution_api_integration/scripts/index.js

```diff
@@ -21,15 +21,19 @@ let grepArgs = [];
 if (type !== 'server') {
   switch (environment) {
     case 'serverlessEnv':
-      grepArgs = ['--grep', '/^(?!.*@brokenInServerless).*@serverless.*/'];
+      grepArgs = ['--grep', '/^(?!.*@skipInServerless).*@serverless.*/'];
       break;
 
     case 'essEnv':
-      grepArgs = ['--grep', '/^(?!.*@brokenInEss).*@ess.*/'];
+      grepArgs = ['--grep', '/^(?!.*@skipInEss).*@ess.*/'];
+      break;
+
+    case 'qaPeriodicEnv':
```

Comment:
Do I get it right all Serverless tests besides skipped ones will run periodically against MKI env?

Reply:
Exactly :)